couler/core/states.py

```diff
@@ -22,6 +25,16 @@
     # set cleanup_proto_workflow to an empty function for compatibility
     cleanup_proto_workflow = lambda: None  # noqa: E731
 
+name_salt = StringGenerator(r"[\c\d]{8}")
+
+
+def default_workflow_name_salter(name):
+    return "{0}-{1}".format(spinalcase(name), name_salt.render())[:62]
```

Comment:
Could you add a comment on why the hardcoded number here?

Reply:
sure, good catch.